app/src/main/java/org/simple/clinic/drugs/EditMedicinesUiRenderer.kt

```diff
@@ -36,6 +36,20 @@ class EditMedicinesUiRenderer(private val ui: EditMedicinesUi) : ViewRenderer<Ed
     val customPrescribedDrugItems = customPrescribedDrugItems(prescribedCustomDrugs)
     val drugsList = (protocolDrugSelectionItems + customPrescribedDrugItems)
         .sortedByDescending { it.prescribedDrug?.updatedAt }
+        .mapIndexed { index, drugListItem ->
+          val isTopItem = index == 0
+          when {
+            isTopItem && drugListItem is ProtocolDrugListItem -> {
+              drugListItem.copy(hasTopCorners = true)
+            }
+            isTopItem && drugListItem is CustomPrescribedDrugListItem -> {
+              drugListItem.copy(hasTopCorners = true)
+            }
+            else -> {
+              drugListItem
+            }
```

Comment:
This logic has a lot of cases, can we update the tests to verify that they are working as expected?

Reply:
I think the only case we haven't covered in the tests is `CustomPrescribedDrugListItem ` as the top item. Can do that. For the rest of them, I have verified they output the correct `hasTopCorners` value.

Reply:
Done. Added the missing case of verifying `CustomPrescribedDrugListItem` as the top item.